app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -281,7 +283,11 @@ const SurveyObservations: React.FC<ISurveyObservationsProps> = (props) => {
       label: 'Mandatory fields have not been filled out'
     },
     recommended: {
-      type: [SUBMISSION_MESSAGE_TYPE.MISSING_RECOMMENDED_HEADER],
+      type: [
+        SUBMISSION_MESSAGE_TYPE.MISSING_RECOMMENDED_HEADER,
+        SUBMISSION_MESSAGE_TYPE.DANGLING_PARENT_CHILD_KEY,
+        SUBMISSION_MESSAGE_TYPE.NON_UNIQUE_KEY
```

Comment:
nit: I think these should be in the mandatory section above.

Reply:
good call, they have been moved